proto/cmp/services/activity/v1alpha/info.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+// ### Activity Services
+//
+// The Activity services are used for both tickets and excursions
+
+package cmp.services.activity.v1alpha;
+
+import "cmp/types/v1alpha/common.proto";
+import "cmp/types/v1alpha/language.proto";
+import "cmp/types/v1alpha/product_code.proto";
+import "cmp/services/activity/v1alpha/activity_types.proto";
+import "google/protobuf/timestamp.proto";
+
+
+message ActivityProductInfoRequest {
+
+  // Message header
+  cmp.types.v1alpha.RequestHeader header = 1;
+
+  // Only respond with the products that are modified after this timestamp
+  google.protobuf.Timestamp modified_after = 2;
+
+  // Languages
+  repeated cmp.types.v1alpha.Language languages = 3;
+
+  // Activity codes
+  repeated cmp.types.v1alpha.SupplierProductCode supplier_codes = 4;
```

Comment:
@SamJaarsma doesn't this cause each activity info request to be supplier specific?

Reply:
No, supplier code mapping is part of all message types (flight, hotel, train,..)